electron-component/scripts/dor-le-dor.js

```diff
@@ -0,0 +1,9 @@
+const {ipcRenderer} = require('electron');
+
+
+function sendMessage(event) {
+    //let topStoryText = document.getElementsByClassName("top-story-text")[0];
+    ipcRenderer.send('asynchronous-message', event.data);
```

Comment:
"asynchronous-message" should be in a consts file and be used in all places you're using it

Reply:
I don't think its a good idea to inject with the script its "consts"